ckanext/canada/recombinant_tables.yaml

```diff
@@ -541,6 +541,157 @@
   default_preview_sort: contract_period_start desc, delivery_date desc
 
 
+- dataset_type: contracts-aggregated
```

Comment:
just a preference, but I'm aligning these types to short all-lowercased names that match our solr cores. How about `contractsa` (like existing `travela`) or `contractsagg` ?